lib/chef/knife/vsphere_vm_clone.rb

```diff
@@ -35,7 +35,7 @@ class Chef::Knife::VsphereVmClone < Chef::Knife::BaseVsphereCommand
 
   option :resource_pool,
          long: '--resource-pool POOL',
-         description: 'The resource pool into which to put the cloned VM'
+         description: 'The resource pool or cluster into which to put the cloned VM'
```

Comment:
Thanks for adding that... We found out that it worked with a cluster mostly by accident :)